superbench/benchmarks/micro_benchmarks/cublas_function.py

```diff
@@ -293,13 +293,15 @@ def _process_raw_result(self, cmd_idx, raw_output):
                     self._curr_run_index, cmd_idx, self._name, raw_output, str(e)
                 )
             )
+            self._result.set_return_code(ReturnCode.MICROBENCHMARK_RESULT_PARSING_FAILURE)
```

Comment:
The common error like "RESULT_PARSING_FAILURE" can be handled by the base class:
https://github.com/microsoft/superbenchmark/blob/40d7905e6b7a551488b47ebac74802880e84ae44/superbench/benchmarks/micro_benchmarks/micro_base.py#L188

For _process_raw_result(), if return False, the base class will set return code as "RESULT_PARSING_FAILURE", no matter what you set it inside the function.

I think only the specific error in _preprocess need to be handled.

Reply:
Same with another PR.

Reply:
I see, thank you for the comment